### ISSUE TITLE
[WebGL] Build failed on Unity 2021.2.0b11

### ISSUE THREAD

**Issue by Meetem** [zappar-xr/universal-ar-unity#3]
WebGL build failed due to errors:
```
Library\Bee\artifacts\WebGL\build\debug_WebGL_wasm\build.js: undefined symbol: zappar_upload_callback_native_dx11 (referenced by top-level compiled C/C++ code)
UnityEngine.GUIUtility:ProcessEvent (int,intptr,bool&)

Library\Bee\artifacts\WebGL\build\debug_WebGL_wasm\build.js: undefined symbol: zappar_upload_callback_native_metal (referenced by top-level compiled C/C++ code)
UnityEngine.GUIUtility:ProcessEvent (int,intptr,bool&)

Library\Bee\artifacts\WebGL\build\debug_WebGL_wasm\build.js: Aborting compilation due to previous errors
UnityEngine.GUIUtility:ProcessEvent (int,intptr,bool&)

Building Library\Bee\artifacts\WebGL\build\debug_WebGL_wasm\build.js failed with output:
emcc:WARNING: --llvm-lto ignored when using llvm backend
```

1. Used UAR version 1.1.1-preview.1
2. With Unity version 2021.2.0b11, without SRP. Build platform is WebGL
3. UAR was installed through package manager, latest version, no other assets was added. Just used scene from examples
4. Execute build, get an error

**Desktop (please complete the following information):**
 - OS: Window 10

Unity 2021.2+ supports ASTC compression which is preferable for mobile devices, so it's better to use it instead other unity versions

**Comment by nfynt-zap** [comment]
@Meetem 

Please make sure to disable the `Development build` option from build settings, before taking the build.

**Comment by Meetem** [comment]
@nfynt-zap This wasn't Development Build.

**Comment by nfynt-zap** [comment]
Thanks for reporting this @Meetem
Indeed with Unity 2021.2.0b11 the WebGL build fails! From the editor player log it seems Unity is using some deprecated emcc arguments and symbols, which results in missing links to certain methods from our lib.

The latest test of the package I did was with the stable version of 2021.1.15f1 and everything was fine there. Further, since this is a beta version of Unity (atm), I am afraid I'll have to deprioritize this issue.

Will investigate this further when I have available bandwidth. Do share your findings if you're investigating the matter.

**Comment by jrDev1** [comment]
> Thanks for reporting this @Meetem Indeed with Unity 2021.2.0b11 the WebGL build fails! From the editor player log it seems Unity is using some deprecated emcc arguments and symbols, which results in missing links to certain methods from our lib.
> 
> The latest test of the package I did was with the stable version of 2021.1.15f1 and everything was fine there. Further, since this is a beta version of Unity (atm), I am afraid I'll have to deprioritize this issue.
> 
> Will investigate this further when I have available bandwidth. Do share your findings if you're investigating the matter.

So I just emailed support about this because 2021.2 is out of Beta and this is the issue I am having. Please fix this!

Thanks,
crafTDev

**Comment by nfynt-zap** [comment]
This should be fixed with new version 2.0.0.
